stubs/Flask-Cors/flask_cors/core.pyi

```diff
@@ -26,21 +26,22 @@ class _Options(TypedDict, total=False):
     always_send: bool | None
 
 LOG: Logger
-ACL_ORIGIN: str
-ACL_METHODS: str
-ACL_ALLOW_HEADERS: str
-ACL_EXPOSE_HEADERS: str
-ACL_CREDENTIALS: str
-ACL_MAX_AGE: str
-ACL_RESPONSE_PRIVATE_NETWORK: str
-ACL_REQUEST_HEADER_PRIVATE_NETWORK: str
-ACL_REQUEST_METHOD: str
-ACL_REQUEST_HEADERS: str
-ALL_METHODS: list[str]
-CONFIG_OPTIONS: list[str]
-FLASK_CORS_EVALUATED: str
-RegexObject: type[Pattern[str]]
-DEFAULT_OPTIONS: _Options
+
+ACL_ORIGIN: Final[str] = "Access-Control-Allow-Origin"
```

Comment:
Nit: If you have a default, you don't need to specify the type:

```python
ACL_ORIGIN: Final= "Access-Control-Allow-Origin"
```
We should really have a flake8 warning for that. I filed PyCQA/flake8-pyi#499.

Reply:
Well, everything will be fine now!